gradlew commands fail with: "failed to find target with hash string 'android-N'"

I have already installed SDK 23 and 24 through Studio, created a working API 24 emulator, and have NDK 12.

Whenever I run a `gradlew` command, e.g.:

    cd tutorial01_load_vulkan
    ./gradlew clean

it fails with:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Exception thrown while executing model rule: NdkComponentModelPlugin.Rules#createNativeLibrary > create(vktuts) > components.vktuts.getBinaries() > withType()
   > Exception thrown while executing model rule: BaseComponentModelPlugin.Rules#createAndroidComponents
      > failed to find target with hash string 'android-N' in: /home/ciro/android-sdk
```

Same error message on SO: http://stackoverflow.com/questions/35937530/android-studio-2-1-preview-failing-to-find-target-with-hash-string-android-n

--- a/common/nativeactivity/build.gradle
+++ b/common/nativeactivity/build.gradle
@@ -6,8 +6,8 @@ def ndkDir = properties.getProperty('ndk.dir')
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
         defaultConfig.with {
             minSdkVersion.apiLevel = 9
             targetSdkVersion.apiLevel = 24

--- a/tutorial01_load_vulkan/app/build.gradle
+++ b/tutorial01_load_vulkan/app/build.gradle
@@ -2,8 +2,8 @@ apply plugin: 'com.android.model.application'
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
 
         defaultConfig {
             applicationId = "com.google.vulkan.tutorials.one"

--- a/tutorial02_prebuild_layers/app/build.gradle
+++ b/tutorial02_prebuild_layers/app/build.gradle
@@ -8,8 +8,8 @@ def valLayerSrc = "${ndkDir}/sources/third_party/vulkan/src"
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
 
         defaultConfig {
             applicationId = "com.vulkan.tutorials.two"

--- a/tutorial03_traceable_layers/app/build.gradle
+++ b/tutorial03_traceable_layers/app/build.gradle
@@ -6,8 +6,8 @@ def ndkDir = properties.getProperty('ndk.dir')
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
 
         defaultConfig {
             applicationId = "com.google.vulkan.tutorials.three"

--- a/tutorial04_first_window/app/build.gradle
+++ b/tutorial04_first_window/app/build.gradle
@@ -2,8 +2,8 @@ apply plugin: 'com.android.model.application'
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
 
         defaultConfig {
             applicationId = "com.google.vulkan.tutorials.four"

--- a/tutorial05_triangle/app/build.gradle
+++ b/tutorial05_triangle/app/build.gradle
@@ -2,8 +2,8 @@ apply plugin: 'com.android.model.application'
 
 model {
     android {
-        compileSdkVersion = "android-N"
-        buildToolsVersion = "23.0.2"
+        compileSdkVersion 24
+        buildToolsVersion "24.0.0"
 
         defaultConfig {
             applicationId = "com.google.vulkan.tutorials.five"

